installscripts/jazz-terraform-unix-noinstances/scripts/configure-cli.sh

```diff
@@ -35,35 +35,35 @@ c = JenkinsLocationConfiguration.get()
 c.url = 'http://$jenkinsurl'
 c.adminAddress = '$email'
 c.save()
-jenkins.model.Jenkins.instance.securityRealm.createAccount("jobexec", "jenkinsadmin")
+jenkins.model.Jenkins.instance.securityRealm.createAccount("jobexec", "$jpassword")
 EOF
 
 if [ "$scm_type" == "gitlab" ]; then
   $jenkins_cli_command groovy = <<EOF
   def cmd = ['/bin/sh',  '-c',  'sed -i "s/ip/$scm_elb/g" $jenkinshome/com.dabsquared.gitlabjenkins.connection.GitLabConnectionConfig.xml']
   cmd.execute()
 EOF
-  ../jenkinscli/credentials/gitlab-user.sh "$jenkins_cli_command" $gitlabuser $gitlabpassword
-  ../jenkinscli/credentials/gitlab-token.sh "$jenkins_cli_command" $gitlabtoken
+  ../jenkinscli/credentials/gitlab-user.sh "$jenkins_cli_command" "$gitlabuser" "$gitlabpassword"
+  ../jenkinscli/credentials/gitlab-token.sh "$jenkins_cli_command" "$gitlabtoken"
 else
-  ../jenkinscli/credentials/bitbucket-creds.sh "$jenkins_cli_command" $bbuser $bbpassword
+  ../jenkinscli/credentials/bitbucket-creds.sh "$jenkins_cli_command" "$bbuser" "$bbpassword"
 fi
 
 #credentials
 ../jenkinscli/credentials/jobexec.sh "$jenkins_cli_command"
-../jenkinscli/credentials/sonar.sh "$jenkins_cli_command" $sonaruser $sonarpassword
-../jenkinscli/credentials/aws.sh "$jenkins_cli_command" $aws_access_key $aws_secret_key
-../jenkinscli/credentials/cognitouser.sh "$jenkins_cli_command" $cognitouser $cognitopassword
+../jenkinscli/credentials/sonar.sh "$jenkins_cli_command" "$sonaruser" "$sonarpassword"
+../jenkinscli/credentials/aws.sh "$jenkins_cli_command" "$aws_access_key" "$aws_secret_key"
+../jenkinscli/credentials/cognitouser.sh "$jenkins_cli_command" "$cognitouser" "$cognitopassword"
 
 #Jobs
-../jenkinscli/jobs/job_create-service.sh "$jenkins_cli_command" $scmpath
-../jenkinscli/jobs/job_delete-service.sh "$jenkins_cli_command" $scmpath
-../jenkinscli/jobs/job_build_pack_api.sh "$jenkins_cli_command" $scmpath
-../jenkinscli/jobs/job_cleanup_cloudfront_distributions.sh "$jenkins_cli_command" $scmpath
-../jenkinscli/jobs/job_build_pack_lambda.sh "$jenkins_cli_command" $scmpath
-../jenkinscli/jobs/job_build_pack_website.sh "$jenkins_cli_command" $scmpath
-../jenkinscli/jobs/job_jazz_ui.sh "$jenkins_cli_command" $scmpath
+../jenkinscli/jobs/job_create-service.sh "$jenkins_cli_command" "$scmpath"
+../jenkinscli/jobs/job_delete-service.sh "$jenkins_cli_command" "$scmpath"
+../jenkinscli/jobs/job_build_pack_api.sh "$jenkins_cli_command" "$scmpath"
+../jenkinscli/jobs/job_cleanup_cloudfront_distributions.sh "$jenkins_cli_command" "$scmpath"
+../jenkinscli/jobs/job_build_pack_lambda.sh "$jenkins_cli_command" "$scmpath"
+../jenkinscli/jobs/job_build_pack_website.sh "$jenkins_cli_command" "$scmpath"
+../jenkinscli/jobs/job_jazz_ui.sh "$jenkins_cli_command" "$scmpath"
 
 #restart
 $jenkins_cli_command restart
-sleep 20 &
+sleep 45 &
```

Comment:
Ideally we should drop this `sleep` as it isn't' reliable and reuse our healthcheck here from outside the script, but for now I will let this pass.